src/viur/core/i18n.py

```diff
@@ -180,8 +180,8 @@ def __init__(
         :param public: Flag for public translations, which can be obtained via /json/_translate/get_public.
         """
         super().__init__()
-
-        self.key = str(key).lower()
+        key = str(key)  # ensure key is a str
```

Comment:
maybe we can add this warning
```suggestion
        if not isinstance(key, str):
            logging.warning(f"Got non-string (type {type(key)}) as {key=}!")
        key = str(key)  # ensure key is a str
```

Reply:
yes please